data/json/items/containers.json

```
@@ -272,6 +272,21 @@
     ],
     "qualities": [ [ "BOIL", 1 ] ]
   },
+  {
+    "id": "bottle_plastic_sealed",
+    "type": "GENERIC",
+    "category": "container",
+    "name": { "str": "sealed plastic bottle" },
```

```suggestion
    "name": { "str": "sealed plastic bottle" },
```
This name won't change after unsealing the bottle, which can be confusing.

---

I think I've fixed it. Changed the open versions to explicitly describe their state.